package.json

```diff
@@ -38,8 +38,11 @@
     "@uppy/aws-s3-multipart": "^3.2.0",
     "@uppy/core": "^3.2.0",
     "@uppy/dashboard": "^3.4.0",
+    "@uppy/drag-drop": "^3.0.2",
```

Comment:
disse andre burde egentlig ikke være nødvendig mener jeg, men legger inn nå for å kunne bygge igjen. Bør revurdere disse senere for nye versjoner av uppy